packages/data-stores/src/site/test/actions.ts

```diff
@@ -425,49 +438,44 @@ describe( 'Site Actions', () => {
 			},
 		} );
 
-		it( 'should not send vertical_id if the design is not verticalizable', () => {
+		it( 'should send style_variation_slug to themes/mine API if the design has style variation', () => {
 			const { setDesignOnSite } = createActions( mockedClientCredentials );
-			const generator = setDesignOnSite( siteSlug, {
-				...mockedDesign,
-				verticalizable: false,
-			} );
+			const generator = setDesignOnSite( siteSlug, mockedDesign, mockedStyleVariation );
 
 			// First iteration: WP_COM_REQUEST to /sites/${ siteSlug }/themes/mine is fired
-			expect( generator.next().value ).toEqual( mockedThemeSwitchApiRequest );
-
-			// Second iteration: WP_COM_REQUEST to /sites/${ siteSlug }/theme-setup is fired
 			expect( generator.next().value ).toEqual(
-				createMockedThemeSetupApiRequest( {
-					trim_content: true,
+				createMockedThemeSwitchApiRequest( {
+					theme: 'zoologist',
+					style_variation_slug: 'default',
+					dont_change_homepage: true,
 				} )
 			);
 		} );
 
-		it( 'should send vertical_id if the design is verticalizable but the vertical id is empty string', () => {
```

Comment:
@arthur791004 -- I removed this test case as the logic does not actually treat empty string and nonempty string differently. Let me know if you think otherwise!

Reply:
I considered this but decided to leave it there. However, I think it's good to remove 👍